test/Datadog.Trace.Tests/SpanTests.cs

```diff
@@ -128,11 +128,11 @@ public void Accurate_Duration(int minimumDurationMilliseconds)
 
             var averageElapsedTime = totalElapsedTime.TotalMilliseconds / iterations;
             var averageSpanTime = totalSpanTime.TotalMilliseconds / iterations;
-            var diff = Math.Abs(averageElapsedTime - minimumDurationMilliseconds);
+            var diff = Math.Abs(averageElapsedTime - minimumDurationMilliseconds); // Should be less than the threshold
```

Comment:
This should be calculating the difference between the expected duration and the span duration.
```suggestion
            var diff = Math.Abs(averageElapsedTime - averageSpanTime); // Should be less than the threshold
```